synapse/app/homeserver.py

```diff
@@ -124,7 +128,9 @@ def create_resource_tree(self, web_client, redirect_root_to_web_client):
             (SERVER_KEY_PREFIX, self.get_resource_for_server_key()),
             (MEDIA_PREFIX, self.get_resource_for_media_repository()),
             (APP_SERVICE_PREFIX, self.get_resource_for_app_services()),
+            (STATIC_PREFIX, self.get_resource_for_static_content())
```

Comment:
Very minor: it's nicer if you have trailing commas in these sorts of lists, as diffs of future additions are more legible.